src/Agent/Widget/ConferenceAgent.vue

```diff
@@ -1,6 +1,11 @@
 <template>
 <b-dropdown text="Agent" variant="outline-primary">
-  <b-dropdown-item v-for="agent in this.$agent.vm.transfer_agents" :key="agent.id" @click="onUpdate(agent.id)">{{ agent.login }}</b-dropdown-item>
+  <div v-if="this.$agent.vm.transfer_agents.length > 0">
```

Comment:
Imo it's better not to display the button at all.

Reply:
I did it as was specified in reach3 issue 44

Reply:
These issues aren't final nor strict demands, but things to think about and to discuss.

Reply:
So, as discussed with @jbrowne3, the button should be disabled if there are no agents and display a popover with explanations on mouse hover.

Reply:
done